android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -1,69 +1,11 @@
 package com.stripeterminalreactnative
 
 import android.graphics.Color
-import com.facebook.react.bridge.ReadableArray
-import com.facebook.react.bridge.ReadableMap
-import com.facebook.react.bridge.WritableArray
-import com.facebook.react.bridge.WritableMap
-import com.facebook.react.bridge.WritableNativeArray
+import com.facebook.react.bridge.*
```

Comment:
nit: undo wildcard import changes and disable wildcard imports in your IDE

Reply:
done